src/server/conn_context.h

```diff
@@ -123,21 +123,28 @@ struct ConnectionState {
     DflyVersion repl_version = DflyVersion::VER0;
   };
 
+  struct SquashingInfo {
+    const ConnectionContext* owner = nullptr;
```

Comment:
Because multiple threads execute commands in parallel during squashing. Added this comment

    // Pointer to the original underlying context of the base command.
    // Only const access it possible for reading from multiple threads,
    // each squashing thread has its own proxy context that contains this info.